packages/cli/src/lib/quire/init-starter.js

```diff
@@ -1,38 +1,56 @@
+import { basename, join, resolve } from 'node:path'
 import { cwd } from 'node:process'
 import fs from 'fs-extra'
 import git from '#src/lib/git/index.js'
-import path from 'node:path'
 /**
  * Clone or copy a Quire starter project
  *
  * @param    {String}   starter   A repository URL or path to local starter
- * @param    {String}   rootPath  Absolute system path to the project root
+ * @param    {String}   projectRoot  Absolute system path to the project root
  * @return   {Promise}
  */
-export async function initStarter (starter, rootPath, quireVersion) {
+export async function initStarter (starter, projectRoot, quire11tyVersion) {
   const remote = 'https://github.com/thegetty/quire-starter-default'
 
   // Clone starter project repository
   await git
-    .cwd(rootPath)
+    .cwd(projectRoot)
     .clone(remote, '.')
     .catch((error) => console.error('[CLI:error] ', error))
 
+  // write project config to project
+  const projectConfig = {
+    projectRoot: resolve(projectRoot),
+    version: quire11tyVersion
+  }
+  const configFilePath = join(projectRoot, 'project.json')
+  const configJSON = JSON.stringify(projectConfig, null, 2)
+  fs.writeFileSync(configFilePath, configJSON)
+
   // Copy 11ty files
-  const fullRootPath = path.resolve(rootPath)
-  const eleventyPath = path.resolve(cwd(), path.join('quire', 'versions', quireVersion))
-  const eleventyFiles = fs.readdirSync(eleventyPath)
+  const fullProjectRootPath = resolve(projectRoot)
+  const eleventyPath = resolve(cwd(), join('quire', 'versions', quire11tyVersion))
+  // @TODO remove `.env` and `_site` from published `quire-11ty` NPM package
+  const eleventyIgnoreFiles = ['.env', '_site']
+  const eleventyFiles = fs
+    .readdirSync(eleventyPath)
+    .filter((filePath) => !eleventyIgnoreFiles.includes(filePath))
 
   // copies all files in `quire/packages/11ty`
   eleventyFiles.forEach((filePath) => {
-    const fileToCopy = path.resolve(eleventyPath, filePath)
-    fs.copySync(fileToCopy, path.join(fullRootPath, path.basename(filePath)))
+    const fileToCopy = resolve(eleventyPath, filePath)
+    if (!eleventyIgnoreFiles.includes(filePath)) {
+      fs.copySync(fileToCopy, join(fullProjectRootPath, basename(filePath)))
+    }
   })
 
   // Reinitialize project as a new git repository
-  await fs.remove(path.join(rootPath, '.git'))
+  await fs.remove(join(projectRoot, '.git'))
 
-  const starterFiles = fs.readdirSync(rootPath)
+  // don't git-add copied `node_modules`
+  const starterFiles = fs
+    .readdirSync(projectRoot)
+    .filter((filePath) => ![...eleventyIgnoreFiles, 'node_modules'].includes(filePath))
```

Comment:
We want to copy the `node_modules` of the installed `@thegetty/quire-11ty` package (for now) into the project, but we don't want to `git add node_modules` during the initial commit. 

**Note:** the `.gitignore` in https://github.com/thegetty/quire-starter-default/ previously included all the 11ty directories (`_includes/`, `_layouts/`, `_plugins/`, `.eleventy.js`) which I commented out for now -- I am assuming this is because our eventual intention is to handle the 11ty pathing stuff in the `pacakges/cli/src/lib/11ty` module so we DON'T need to manually copy 11ty files into a created project